test/object-store/realm.cpp

```diff
@@ -548,6 +538,57 @@ TEST_CASE("Get Realm using Async Open", "[asyncOpen]") {
         REQUIRE(called);
     }
 
+    SECTION("can write a realm file without client file id") {
```

Comment:
Love how descriptive this test is. I can actually read the comments only (ignore the code) and quickly understand what the test is doing.